haiku-apps/vwget/vwget-20141229.recipe

```diff
@@ -5,8 +5,11 @@ it as a normal app and select the URL and destination in the GUI."
 HOMEPAGE="https://github.com/HaikuArchives/VWGet"
 COPYRIGHT="1999 Santiago Lema"
 LICENSE="GNU GPL v2"
-REVISION="1"
-SOURCE_URI="git+https://github.com/HaikuArchives/VWGet.git#29ff74d068526c01edf7e1c0ef887a8f4fac252f"
+REVISION="2"
+COMMIT="29ff74d068526c01edf7e1c0ef887a8f4fac252f"
+SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="bcbccd678e46d061e40f5fbc40ff7d2c99ed1e74f3439f3725aae8b5650a2ab5"
+SOURCE_DIR="vwget-20141229-$COMMIT"
```

Comment:
I would replace
```
REVISION="2"
COMMIT="29ff74d068526c01edf7e1c0ef887a8f4fac252f"
SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
CHECKSUM_SHA256="bcbccd678e46d061e40f5fbc40ff7d2c99ed1e74f3439f3725aae8b5650a2ab5"
SOURCE_DIR="vwget-20141229-$COMMIT"
```
by
```
REVISION="1"
srcGitRev="29ff74d068526c01edf7e1c0ef887a8f4fac252f"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="bcbccd678e46d061e40f5fbc40ff7d2c99ed1e74f3439f3725aae8b5650a2ab5"
SOURCE_DIR="vwget-20141229-$srcGitRev"
```
That's because srcGitRev is imho a slightly better choice than COMMIT, since:
* there are already 36 recipes using "srcGitRev", compared to only 4 which are using "COMMIT";
* contrary to "COMMIT", which looks like a reserved HaikuPorter variable, "srcGitRev" makes it clear that it is not a reserved but a custom variable.

And since the resulting SOURCE_URI is unchanged we don't really need to bump REVISION.

But if another contributor says that COMMIT is ok and/or that we should bump REVISION then that's OK for me too ;-)